src/layout/section/heroSection/index.tsx

```diff
@@ -1,123 +1,168 @@
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 import { responsiveBreakpoints, componentSizes, color, space } from '_utils/branding'
+import HeroSection, { HeroSectionProps } from './heroSection'
 
-import HeroSection from './heroSection'
+// Remove these deprecated styles once all the button content parts have been removed.
+const DEPRECATED_STYLES = css`
+  & {
+    justify-content: flex-end;
+  }
+
+  & .hero-info {
+    padding-top: 0;
+    padding-bottom: ${space.xl};
+    margin-bottom: ${space.xl};
+  }
+`
+
+const DEPRECATED_SMALL_MEDIA_STYLES = css`
+  & {
+    margin-bottom: 0;
+    height: initial;
+  }
+
+  & .hero-image {
+    position: static;
+    height: 136px;
+    background-image: ${(props: HeroSectionProps) => `url(${props.heroImageUrl})`};
+  }
+
+  & .hero-image::after {
+    content: none;
+  }
+
+  & .hero-info {
+    padding-top: ${space.xl};
+  }
+
+  & .hero-info-title {
+    color: ${color.primaryText};
+  }
+
+  & .hero-info-text {
+    color: ${color.secondaryText};
+    font-size: 16px;
+  }
+
+  & .hero-content {
+    align-items: center;
+  }
+`
 
 const StyledHeroSection = styled(HeroSection)`
   & {
     background-color: ${color.defaultBackground};
     position: relative;
     z-index: 1;
-    padding: 0;
+    height: 75vh;
+    margin-bottom: ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
   }
 
   & .hero-image {
-    position: relative;
-    min-height: 136px;
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
     background-size: cover;
     background-position: center;
-    background-image: ${props => `url(${props.heroImageUrl})`};
+    background-image: ${props => `url(${props.heroImageUrlLarge})`};
+
+    /* Hide the shadow. */
+    overflow: hidden;
+  }
+
+  & .hero-image::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 75%;
+    display: block;
+    background: linear-gradient(
+      0deg,
+      rgba(0, 0, 0, 0.4) 72%,
+      rgba(0, 0, 0, 0.2) 85%,
+      rgba(0, 0, 0, 0) 100%
+    );
   }
 
   & .hero-info {
-    text-align: center;
-    margin-left: auto;
-    margin-right: auto;
+    width: 100%;
     max-width: ${componentSizes.largeSectionWidth};
-    padding: ${space.xl};
-    padding-top: 0;
+    padding: 0 ${space.xl};
+    padding-top: 120px;
```

Comment:
Where this magic number comes from?

Reply:
[Looking at the design](https://sketch.cloud/s/2bKDG/a/Q5Y2Em#Inspector), the content is not vertically centered, it's a bit lower so the woman's face isn't hidden by the text, this padding shifts the content vertically to fit the design (in 1440 x 960) and other sizes that are not represented by the design but still exist.

This is clearly not perfect, if you have a better suggestion I'll take it 😁.